docs/ansible.md

```diff
@@ -0,0 +1,28 @@
+# Ansible playbooks
+
+## Executive Summary
+As part of our tasks, we aimed to create virtual machines (VMs) to facilitate the work of developers. The following needs were mentioned:
+
+- Testing applications before requesting IT to install the software on our personal laptops
+- Facilitating the work of certain developers (e.g., having CLIs already installed and configured)
+To set up virtual machines in the cloud, using Ansible was very useful to enable scaling in case we need more VMs in the future.
+
+## Glossary
+
+**Ansible:** Ansible is an open-source software tool for IT automation. It automates the provisioning, configuration, and deployment of servers and applications. Ansible is agentless, meaning it does not require any software to be installed on the machines it manages. It uses SSH to connect to machines and execute commands.
```

Comment:
is there no line length checks? this is way >80 chars?

Reply:
Hmmmmmmm

Reply:
Done. About the markdown, I think jonathan is working on it inside this PR https://github.com/ai-cfia/github-workflows/pull/111